Example/Podfile.lock

```diff
@@ -37,9 +37,9 @@ SPEC CHECKSUMS:
   Expecta: 3b6bd90a64b9a1dcb0b70aa0e10a7f8f631667d5
   OCHamcrest: b284c9592c28c1e4025a8542e67ea41a635d0d73
   OCMockito: 677cbb4a18fd492b5a4fb10144dada4de5ddb877
-  Segment-ComScore: 8ae19e5fc419df807ada8226c917e48c2a1f77f4
+  Segment-ComScore: 711dfbc951b5fbe081472e4cb684335d9532f307
   Specta: 3e1bd89c3517421982dc4d1c992503e48bd5fe66
 
 PODFILE CHECKSUM: 58708c888acf3799898d9149cc9266ed0823f288
 
-COCOAPODS: 1.8.4
+COCOAPODS: 1.7.0.beta.3
```

Comment:
Did we need to drop the version?

Reply:
I will update the cocoa pods version of my local and push the code

Reply:
Updated Cocoapods to latest stable version available `https://github.com/CocoaPods/CocoaPods/releases`

Reply:
I have reverted this to 1.8.4